src/utils/graphql/queries/getResponsesInApplication.query.ts

```diff
@@ -0,0 +1,15 @@
+import { gql } from '@apollo/client'
```

Comment:
Is is necessary to make a new query for this? It seems very similar to `getElementsAndResponses`. Can you not just use that?

Reply:
Thanks! Done: 93d4720c8b79c37c181ce8b2d466825030d3db46